index.html

```diff
@@ -34,8 +34,7 @@
         <a href="apizoo/">API Zoo Index</a>
     </div>
     <div class="highlight-clean" style="padding-bottom: 10px;">
-            <h1 class="text-center"> 🦍 Gorilla: Large Language Model Connected with Massive APIs</h1>
-        <div><p></p></div>
+        <h1 class="text-center" style="margin-left: 10px; margin-right: 10px;"> 🦍 Gorilla: Large Language Model Connected with Massive APIs</h1>
```

Comment:
This style is redundant when you already have the `!important` thing. Please try to only include the code that's needed.